03-rnaseq/ortholog-mapping_rnaseq_01_ensembl.Rmd

```diff
@@ -305,49 +305,50 @@ We don't want to handle duplicate data, so let's remove those duplicates before
 
 ```{r}
 human_mouse_key <- human_mouse_key %>%
-  # We need to use the `distinct()` function to remove duplicates resulted from
-  # ignoring the additional columns in the `mouse` object
+  # Use the `distinct()` function to remove duplicates resulting from
+  # dropping the additional columns in the `mouse` data frame
   dplyr::distinct()
 ```
 
 Now let's join the mapped data from `human_mouse_key` with the gene data in `mouse_genes`.
 
 ```{r}
 # First, we need to convert our vector of mouse genes into a data frame
-human_mouse_mapped_df <- data.frame("Gene" = mouse_genes) %>%
+human_mouse_mapped_df <- mouse_genes %>%
   # Now we can join the mapped data
   dplyr::left_join(human_mouse_key, by = c("Gene" = "mouse_ensembl"))
 ```
 
 Here's what the new data frame looks like:
 
 ```{r rownames.print = FALSE}
-head(human_mouse_mapped_df, n = 25)
+head(human_mouse_mapped_df, n = 10)
 ```
 
+Looks like we have mapped IDs!
+
+So now we have all the mouse genes mapped to human, but there might be places where there are multiple mouse genes that are orthologous to the same human gene, or vice versa.
+
 Let's get a summary of the Ensembl IDs returned in the `human_ensembl` column of our mapped data frame.
 
 ```{r}
-# We can use this `count()` function after `group_by()`to get a count of how many
+# We can use this `count()` function to get a tally of how many
 # `mouse_ensembl` IDs there are per `human_ensembl` IDs
 human_mouse_mapped_df %>%
-  dplyr::group_by(human_ensembl) %>%
-  dplyr::count() %>%
-  # Sort by highest `n` which would be the human Ensembl ID with the most mapped
+  # count the number of rows per human gene
+  dplyr::count(human_ensembl) %>%
+  # Sort by highest `n` which will be the human Ensembl ID with the most mapped
   # mouse Ensembl IDs
   dplyr::arrange(desc(n))
 ```
 
-Looks like we have mapped IDs!
+There are certainly a good number of places where we mapped multiple zebrafish Ensembl IDs to the same human symbol!
+We'll look at this in a bit.
 
-Now, let's get an idea of how many mouse Ensembl IDs we have that were not mapped to human Ensembl IDs.
+We can also see that there 19,126 mouse Ensembl IDs that did not map to a human Ensembl ID.
+These are the rows with a value of NA.
+This seems like a lot, but many of these may not be "real" genes, and we  do not expect everything to map neatly across species.
```

Comment:
```suggestion
This seems like a lot, but many of these may not be "real" genes, and we do not expect everything to map neatly across species.
```

Reply:
What's a "real" gene anyway? I there a different way we can phrase this to be more specific/informative?

Reply:
This is not a silly comment! I'm trying to convey that many of the things we annotate are not all that well supported, so the fact that we are missing 19K genes is more likely the result of spurious annotations than error. I think having a reference for this claim might be useful. I'll look for one.

Reply:
I forgot I had one un-silly comment.